controllers/userController.js

```diff
@@ -1,15 +1,31 @@
-const admin = require('firebase-admin');
+const { auth, storage } = require('firebase-admin');
+const { fireStore } = require("../config/firebaseConfig");
 const signUp = async (req, res) => {
-  const { email, password } = req.body;
+  const { email, password, confirmPassword, address, firstName, lastName, profileImage } = req.body;
   if (!email || !password) {
     res.status(404).send({ message: "Please fill all the required fields" })
   }
+  if (password != confirmPassword) {
+    res.status(401).send({ message: "Password and confirmPassword should be same" })
+  }
   try {
-    const user = await admin.auth().createUser({
+    const user = await auth().createUser({
       email,
       password,
     });
-    res.status(200).send(user)
+    fireStore.collection("users").doc(user.uid).set({
+      firstName,
+      address,
+      lastName,
+    });
+    const uploadProfilePhoto = async (file, filename) => {
```

Comment:
move this code to a saparete file, so any function can use this @ashok-choudhary-0